### ISSUE TITLE
support pasting multiline text as multiple

### ISSUE THREAD

**Issue by WebFreak001** [lolo-io/OneList#23]
When I get a checklist sent by other people via a text manager it is usually some kind of formatted list like

```
o hello
x world
```
or
```
[ ] potato
[ ] apple
[ ] chocolate
```

I would like to have the option to either have all line separated texts, pasted into the entry field, be treated as separate todo list items or some kind of parser for it.

Currently when pasting multiline text it renders it all as a single line in the input and becomes a huge entry when adding.

**Comment by crumblingstatue** [comment]
A family member sent me a shopping list in SMS and I wanted to convert it to a todo list.
I think it would be nice to have an "import" feature where you can split on either newlines or commas, etc.

**Comment by mediapinta** [comment]
Yes, please! I have very long lists (movies, books, series, comics...) and would be great just paste them to create a list in the app. One line = one new item.

Another option could be to have numbered lists and the possibility of adding numbered items.

Ex. list:

1 item
2 item2
3 item 3

new item: 4 item new

updated list:

1 item
2 item2
3 item 3
4 item new

new item: 3 item5

1 item
2 item2
3 item5
4 item 3
5 item new


Thank you for this simple and great app!

Greetings,
ifrit
